kpm/lib/kpm/maven_helper/maven_facade.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative 'maven_actions'
+
+module KPM
+  module MavenFacade
+    ArtifactMalformedException = MavenFacade::ArtifactMalformedException
+    UnexpectedStatusCodeException = MavenFacade::UnexpectedStatusCodeException
+    
+    # Factory method for creating Maven remote connections
```

Comment:
You don't need a new factory, just a new `MavenCentralApiCalls` action that implements the client to Maven Central?